nixos/modules/services/web-apps/freshrss.nix

```diff
@@ -243,6 +244,9 @@ in
         in
         {
           description = "Set up the state directory for FreshRSS before use";
+          wants = [ "network-online.target" ];
+          after = [ "network-online.target" ];
+          bindsTo = [ ] ++ optional usePostgresql "postgresql.service" ++ optional useMysql "mysql.service";
```

Comment:
It seems systemd.services.freshrss-config does not need a database (I am not sure though).  If so, this line is not needed.

Reply:
In my testing it failed without a database, hence I added these lines